demo/alloy-showcase-webapp/src/test/java/com/liferay/faces/test/alloy/showcase/datatable/DataTableTesterBase.java

```diff
@@ -238,37 +184,31 @@ protected String getRowSelectionInputXpath(Customer customer) {
 			"')]//input[contains(@id,'customers:')]";
 	}
 
-	/**
-	 * Determines whether or not the specified page number is active in the rendered <code>alloy:paginator</code>
-	 * markup.
-	 */
-	protected ExpectedCondition<Boolean> isPaginatorPageNumberButtonActive(int pageButtonNumber) {
```

Comment:
Local variable compromise commits, without the word "is":
https://github.com/liferay/liferay-faces-alloy/commit/2150e3e11d30e30cf98515d1d475187cb468bda5
https://github.com/liferay/liferay-faces-portal/commit/44a59da8db94df42b69731be52746a46dc211983